backend/src/flotilla/services/isar/requests.py

```diff
@@ -0,0 +1,53 @@
+from abc import ABCMeta, abstractmethod
+
+from azure.identity import DefaultAzureCredential
+from requests import Response
+
+from flotilla.services import RequestHandler, get_azure_credentials
+from flotilla.settings import settings
+
+
+class IsarRequestsInteface(metaclass=ABCMeta):
+    @abstractmethod
+    def start_mission(self, host: str, port: int, mission_id: int) -> Response:
+        pass
+
+    @abstractmethod
+    def stop(self, host: str, port: int) -> Response:
+        pass
+
+
+class IsarRequests(IsarRequestsInteface):
```

Comment:
Sounds like a list of requests. Is this not just an API? Also a typo in "interface"
```suggestion
class IsarAPI(IsarAPIInterface):
```

Reply:
Hmm, don't know if feel like API is any better